packages/next/src/client/components/react-dev-overlay/internal/helpers/group-stack-frames-by-module.ts

```diff
@@ -47,21 +40,23 @@ function getFramework(
  * > react
  *
  */
-export function groupStackFramesByFramework(
+export function groupStackFramesByModule(
   stackFrames: OriginalStackFrame[]
 ): StackFramesGroup[] {
   const stackFramesGroupedByFramework: StackFramesGroup[] = []
 
   for (const stackFrame of stackFrames) {
     const currentGroup =
       stackFramesGroupedByFramework[stackFramesGroupedByFramework.length - 1]
-    const framework = getFramework(stackFrame.sourcePackage)
+    // TODO: should come from react-dev-overlay/server/middleware.ts
+    // const framework = stackFrame.sourcePackage
```

Comment:
is this commented variable needed? not sure what the TODO comment mean

Reply:
Same as https://github.com/vercel/next.js/pull/62206#discussion_r1493547625